web/packages/teleport/src/Navigation/SideNavigation/Search.tsx

```diff
@@ -89,10 +78,23 @@ export function SearchSection({
         id={`panel-${section.category}`}
         onFocus={() => handleSetExpandedSection(section)}
       >
-        <SearchContent
-          navigationSections={searchableNavSections}
-          currentView={currentView}
-        />
+        <Box
+          css={`
+            overflow-y: auto;
+            padding: 3px;
```

Comment:
Why 3px? Should we use the theme's spacing instead?

Reply:
This is to prevent a visual bug where the outline around a subsection in focus mode would get clipped if there wasn't enough padding. I checked in with Kenny and he suggested this.